### ISSUE TITLE
Invalid URL causes error

### ISSUE THREAD

**Issue by treyhuffine** [alanshaw/embed-video#14]
When the the URL is invalid, we the error `Uncaught TypeError: Cannot read property 'indexOf' of null`. I'll add a fix for the issue.

**Comment by alanshaw** [comment]
Yes there should be a try/catch around `URL.parse` that'll call the callback if an error is caught.
